src/sidebar/store/modules/sidebar-panels.js

```diff
@@ -91,7 +91,7 @@ function closeSidebarPanel(panelName) {
  * designated `panelState`
  *
  * @param {string} panelName
- * @param {boolean} panelState - Should the panel be active?
+ * @param {boolean} [panelState] - Should the panel be active?
```

Comment:
This means "toggle current state" if omitted? If so I think that is worth mentioning in the prop docs.